mongo/client_encryption.go

```diff
@@ -37,6 +38,7 @@ func NewClientEncryption(keyVaultClient *Client, opts ...options.Lister[options.
 
 	ce := &ClientEncryption{
 		keyVaultClient: keyVaultClient,
+		closed:         false,
```

Comment:
The [zero value](https://go.dev/ref/spec#The_zero_value) for booleans in Go is `false`. Suggest removing this line.